.travis.yml

```diff
@@ -5,6 +5,31 @@ env:
     - CONFIG=opt
     - TEST=objc
     - JOBS=1
+  matrix:
+    - TEST_PATH="src/objective-c/tests" WORKSPACE="Tests.xcworkspace"
+      SCHEME="RxLibraryUnitTests" BUILD_ONLY="false" INTEROP_SERVER="false"
```

Comment:
nit: It'd be nice if you reordered the variables so `SCHEME` is first, followed by `WORKSPACE`. That way it's easier to tell which test is which from the Travis page (see https://travis-ci.org/grpc/grpc/builds/140600382 ).